packages/trader/src/App/app.jsx

```diff
@@ -29,6 +29,10 @@ class App extends React.Component {
         this.root_store.ui.setPromptHandler(false);
     }
 
+    componentDidMount() {
+        this.root_store.ui.changeTheme();
```

Comment:
Instead of adding these separately in Trader and Bot, is it possible to do it in `packages/core/src/Stores/common-store.js`? Check if platform changes and update the theme. You can use a reaction for this.

Reply:
Nice idea, let me check it.

Reply:
@carol-binary Since the problem with `routing_history` fixed, `changeTheme()` will return the correct value when running. I added those method calls before fixing `routing_history` problem. So no need to calling this method anymore.